File: ServiceManagerFrontEnd/src/core/commons/ErrorCodes.ts
export default class ErrorCodes {
  static None = 0;
  static InternalServerError = 500;
  static NotImplemented = 501;
  // system wise error codes start with 1000
  static DataExist = 1000;
  // use error codes start with 2000
  static UserAndPasswordNotMatch = 2000;
}

File: ServiceManagerBackEnd/Commons/ErrorCodes.cs
namespace ServiceManagerBackEnd.Commons;

public static class ErrorCodes
{
    public const int None = 0;
    public const int InternalServerError = 500;
    public const int NotImplemented = 501;
    // system wise error codes start with 1000
    public const int DataExist = 1000;
    // user error codes start with 2000
    public const int UserAndPasswordNotMatch = 2000;
    public const int TokenInvalid = 2001;
}